### ISSUE TITLE
Add option to disable truecaller listener once done

### ISSUE THREAD

**Issue by HishamMubarak** [aaronalphonso/react-native-truecaller-sdk#8]
`TRUECALLER.on(TRUECALLER_EVENT.TrueProfileResponse, profile => { //callback executed })`

Once the Truecaller listener is added using the above code, in the componentDidMount, there are no methods available to remove this event listener. There after in the entire app session until exiting, whenever the app state goes to background and returns, this callback function gets called.

I guess there should be some method like `TRUECALLER.removeListener(TRUECALLER_EVENT.TrueProfileResponse)` to remove this listener, so we can invoke that in componentWillUnmount or whenever the verification with truecaller is done.

**Comment by kindacoder** [comment]
@HishamMubarak  did you find some way to fix this ?

**Comment by HishamMubarak** [comment]
> @HishamMubarak did you find some way to fix this ?

Nope, haven't  yet figured out a way to solve this. Ignored it for now. Will need to add a method to remove the listener and submit and PR

**Comment by swapnil001** [comment]
Hi,

Any update on the remove listener PR?
